schema.graphql

```diff
@@ -485,11 +544,78 @@ type MatchQuestionBuzz {
 """
 Docs group: Match
 """
-type MatchQuestionBonus {
+type BuzzPosition {
+  """
+  The time of the buzz. An ISO 8601-formatted string, with time zone, in the tournament's local time zone. May not be before the start time of reading the relevant question.
+  """
+  time: Date
+
+  """
+  The (`0`-based) index of the word on which the player buzzed.
   """
-  Which bonus number this was. May or may not match the `question_number` of the `MatchQuestion` parent, depending on whether bonuses are paired with tossups or simply read in order (and if the latter, on whether any bonus-earning questions failed to lead to a bonus).
+  word_index: Int
+
+  """
+  The (`0`-based) index of the clue on which the player buzzed. Clues may be defined in the `Question` object.
+  """
+  clue_index: Int
+}
+
+"""
+This object can be reused for every time the same question is read, i.e., it represents a question in general rather than a question being read in a specific match.
+
+Docs group: Match
+"""
+type Question {
+  """
+  The number of the question in the packet.
   """
   question_number: Int!
+  
+  """
+  The type of the question. We generally expect `tossup` or `bonus`, but are not restricting the values since some formats may use other types of questions.
+  """
+  type: String!
+  
+  """
+  The number of parts the question has (typically `1` if `type = "tossup"` and `3` if `type = "bonus"`, but some formats may vary).
+  """
+  parts: Int
+  
+  """
+  The identifier for the question in some external system, if any (e.g., the question IDs that NAQT prints below each question in its packet).
+  """
+  external_id: String
+  
+  """
+  Freeform information about the question's category, e.g. `British History`.
+  """
+  category: String
+  
+  """
+  Freeform information about the question's category at a higher level of aggregation, e.g. `History`.
+  """
+  category_group: String
+  
+  """
+  The text of the question (as a single field).
+  """
+  text: String
+  
+  """
+  The text of the question, broken up into clues so that `BuzzPosition`s can reference which clue was being buzzed on. Defining "clue" is outside the scope of this schema.
+  """
+  clues: [String]
```

Comment:
Just throwing this out there without an opinion, but would we do better with a `QuestionClue` object with a `word_index` field and potentially a `value` field for an archaic bonus question? Is there something to be gained by combining the concepts of "parts" and "clues"?

Reply:
> Just throwing this out there without an opinion, but would we do better with a `QuestionClue` object with a `word_index` field and potentially a `value` field for an archaic bonus question?

Can you explain "`value` field for an archaic bonus question"? Are you thinking of bonuses with different parts worth different numbers of points?


> Is there something to be gained by combining the concepts of "parts" and "clues"?

These feel pretty different to me—clues are mutually exclusive in terms of answering, while bonus parts are additive. I agree they are structurally similar, but I think the semantic difference is considerable.

Reply:
> Are you thinking of bonuses with different parts worth different numbers of points?

Yes.

> These feel pretty different to me—clues are mutually exclusive in terms of answering, while bonus parts are additive. I agree they are structurally similar, but I think the semantic difference is considerable.

Okay.